TrackletAlgorithm/Constants.h

```diff
@@ -4,8 +4,8 @@
 #include "ap_int.h"
 
 #ifdef CMSSW_GIT_HASH
-  #include "Settings.h"
-  using namespace trklet;
+#include "L1Trigger/TrackFindingTracklet/interface/Settings.h"
+using namespace trklet;
```

Comment:
During CMSSW code reviews, we're told never to put "using namespace" in a header file. Such statements must only appear in .cc files, since otherwise they can affect multiple files.

Reply:
The namespace line was pushed by someone else, but I can remove it if that's appropriate to do for this issue.

Reply:
It should be removed, but you may then have then add it to .cc files to make the code compile in CMSSW.